LandSandBoat/server
# Implement Chigoe "Pets" on ToAU Mobs

**_I affirm:_**

## What does this pull request do?
Implement Chigoe "pets" for ToAU mobs. 

Source: https://www.bg-wiki.com/ffxi/Category:Chigoe

**Features**

- A chigoe will spawn the each time the defined mob uses a mobskill (up to 5 times)
- Chigoe will get one-shot with a critical hit
- Chigoe will get one-shot when hit with a pre-defined list of job abilities (bashes, etc)
- Chigoe will get one-shot when hit with a weaponskill

## Steps to test these changes

Visit any of the affected areas and find a mob that spawns chigoes.
!tp 3000 and notice it will spawn a chigoe up to 5 times.
You should be able to one shot them as described above.
